snapcraft/README

```diff
@@ -0,0 +1,26 @@
+This folder contains the files needed to build a snap package for nikola.
+
+To build, in Ubuntu 16.04:
+
+apt install snapd snapcraft
+snap login some.account@whatever.com
+snapcraft
+cp nikola.py stage/usr/bin/nikola
+snapcraft
+
+To just use it:
```

Comment:
Yes, currently it's just a draft so I can keep track of where I am with
this.

On Fri, Jun 17, 2016 at 11:20 AM Chris Warrick <notifications@github.com>
wrote:

> In snapcraft/README
> <https://github.com/getnikola/nikola/pull/2375#discussion_r67517386>:
>
> > @@ -0,0 +1,26 @@
> > +This folder contains the files needed to build a snap package for nikola.
> > +
> > +To build, in Ubuntu 16.04:
> > +
> > +apt install snapd snapcraft
> > +snap login some.account@whatever.com
> > +snapcraft
> > +cp nikola.py stage/usr/bin/nikola
> > +snapcraft
> > +
> > +To just use it:
>
> This should live in our getting started document when snaps actually work
> (i.e. you can build the demo site correctly). We should also mention other
> distros, and provide a link to snapcraft.io.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/getnikola/nikola/pull/2375/files/f3180581af5467538aa95033ff4caf0ccdef5bf8#r67517386>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAGK35vi6g0EFsZ8Vs9E85HYw8ygptDks5qMq0WgaJpZM4I27fk>
> .
>